recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -1,17 +1,16 @@
-BRANCH = "indirect_skl"
-SRC_URI:remove = " ${GNU_MIRROR}/grub/grub-${PV}.tar.gz"
-SRC_URI:append = " git://github.com/3mdeb/grub.git;branch=${BRANCH};protocol=https"
-
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI += " \
+SRC_URI = " \
+    git://github.com/3mdeb/grub.git;branch=${BRANCH};protocol=https \
```

Comment:
use tb repo

Reply:
@macpijan done in [1927317](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/19273177e3ba6311b82834e423852ab4399750c3)